3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -27,9 +27,9 @@ exercise file.
 const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
-  // TODO Refactor this function
   const dice = [1, 2, 3, 4, 5];
-  return rollDie(1);
+  const promises = dice.map((die) => rollDie(die));
```

Comment:
While your code is fine, this can be shortened to:
 
```suggestion
  const promises = dice.map(rollDie);
```
The `rollDie()` function takes one argument. Your arrow function simply passes on the `die` argument to the `rollDie()` function. You might as well cut out the middle man (the arrow function) and let `.map()` directly call `rollDie`.